src/Backend/Core/Engine/DataGridFunctions.php

```diff
@@ -19,6 +19,8 @@
  */
 class DataGridFunctions
 {
+    protected static $dataGridUsers;
```

Comment:
you should make this an array, since you'll be adding items to it in the getUser method.